### PR TITLE
Added freeClaim property on the networks and updated the docs template

### PR DESCRIPTION
**Pull request by wonderwomancode**
Making it clear that we have free claims and which networks they are on.

### CODE REVIEW

**Review by julien51 — COMMENTED** [review]
Could you please also update the locksmith file so we use the values set here ?

**Review by julien51 — COMMENTED** [review]
This is a good start though!

File: packages/networks/src/networks/avalanche.ts [hunk 1]
```diff
@@ -41,6 +41,7 @@ export const avalanche: NetworkConfig = {
   startBlock: 17188332,
   previousDeploys: [],
   isTestNetwork: false,
+  freeClaim: 1,
```
Comment (julien51) [inline packages/networks/src/networks/avalanche.ts:44]:
```suggestion
  maxFreeClaimCost: 1,
```

**Comment by wonderwomancode** [comment]
> Do you need help with the TS issue @wonderwomancode ?

Yes, now I'm not seeing why is giving me an error now that maxFreeClaimCost does not exist after I fixed the renamed ones I missed and added it to the NetworkConfig type definition.

**Pull request merged**